src/EnergyPlus/CondenserLoopTowers.cc

```diff
@@ -1145,9 +1145,11 @@ namespace CondenserLoopTowers {
 					ShowSevereError( cCurrentModuleObject + " \"" + SimpleTower( TowerNum ).Name + "\". Low-speed nominal capacity must be less than the high-speed nominal capacity." );
 					ErrorsFound = true;
 				}
-				if ( SimpleTower( TowerNum ).TowerFreeConvNomCap >= SimpleTower( TowerNum ).TowerLowSpeedNomCap ) {
-					ShowSevereError( cCurrentModuleObject + " \"" + SimpleTower( TowerNum ).Name + "\". Free convection nominal capacity must be less than the low-speed nominal capacity." );
-					ErrorsFound = true;
+				if ( !SimpleTower( TowerNum ).TowerLowSpeedNomCapWasAutoSized ) {
+					if ( SimpleTower( TowerNum ).TowerFreeConvNomCap >= SimpleTower( TowerNum ).TowerLowSpeedNomCap ) {
+						ShowSevereError( cCurrentModuleObject + " \"" + SimpleTower( TowerNum ).Name + "\". Free convection nominal capacity must be less than the low-speed nominal capacity." );
+						ErrorsFound = true;
+					}
```

Comment:
This is fine as long as you are confident that the autosized low speed nominal capacity will definitely be higher than the free convection nominal capacity.  I haven't dug in that code in a while, is that a guarantee?  Or does this warning need to be moved into the sizing function there?